rcl/test/rcl/client_fixture.cpp

```diff
@@ -158,6 +158,8 @@ int main(int argc, char ** argv)
 
     // Initialize a request.
     test_msgs__srv__Primitives_Request client_request;
+    // TODO zero initialization necessary until https://github.com/ros2/ros2/issues/397 is implemented
```

Comment:
arg this is both too long and missing a name attached to the TODO, I expect cpplint to not be happy :/

Reply:
Fixed in e322f7daa8c2e74fc8402f45c1bc80aa790c3bd3: [![Build Status](https://ci.ros2.org/buildStatus/icon?job=ci_linux&build=5044)](https://ci.ros2.org/job/ci_linux/5044/)